### DISCUSSION TITLE
Rotate Map From Bottom Center Point

### DISCUSSION THREAD

**Discussion by lgmadurange** [maplibre/maplibre-gl-js#1989] [Q&A] [answered]
I Want To Rotate The Map From Bottom Point, Currently I Can Rotate The Map Around Center Point.

Current Rotation Way---
![current rotation](https://user-images.githubusercontent.com/53076336/209475345-30f6df6a-779f-457c-94e5-f24e7a4e2076.png)

Required Rotation Way---
Rotating The Map From The Bottom Point ---
![required rotation](https://user-images.githubusercontent.com/53076336/209475355-4c3102b4-011c-436b-863d-566d865b2b1c.png)

**Comment by wipfli** [comment]
That would be a cool feature to have I think.

**Comment by HarelM** [comment, marked as answer]
There is an offset parameter in easeTo, maybe it will allow this behavior.

**Comment by LittleNoobCN** [comment]
您好，我已收到您的邮件，谢谢！
